README.md

```diff
@@ -4,27 +4,30 @@
 
 Malasakit is a customizable participatory assessment platform that collects and integrates quantitative assessment, qualitative feedback, and peer-to-peer collaborative filtering on ways local communities can become better prepared for typhoons and floods. 
 
-## Requirements:
 
+## Setup (if using pip):
 - python 2.7
-  - comes by default on most systems, otherwise use [this tutorial on virtual environments.](http://justinmi.me/blog/2017/04/15/python-virtual-env)
+  - comes by default on most systems, otherwise use 
   - check installed version with `python -V` on terminal
 - pip: package manager for python
   - install with `sudo easy_install pip`
```

Comment:
Minor nitpick, but using `easy_install` to install the `pip` package seems to use circular logic. It would be better to defer to `pip`'s official documentation on installation.